README.md

```diff
@@ -156,6 +156,7 @@ have already been built into our Foundation library. We use
 to display components outside of the app context. To open it, run:
 
 ```sh
```

Comment:
added this since this seemed like a natural place to get to know the project & doesn't work without knowing you need to run yarn install. i guess i could have guessed because of the npm workflow, but thought i'd remove any friction for the next rube like me. i added "install" so people understand the purpose. even dummies like me :)